### ISSUE TITLE
Unable to install Quire v1.0.0-rc.3 Windows 11

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#672]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

Run `npm install -g @thegetty/quire-cli` on a Windows 11 machine, and then when the install is complete, run `quire --version` to verify.

### Actual behavior

This error is thrown, and is the same with any `quire` command, not just `--version`:

```
PS C:\WINDOWS\system32> quire --version
& : The term '-S.exe' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\geeal\AppData\Roaming\npm\quire.ps1:24 char:7
+     & "-S$exe" node --no-warnings "$basedir/node_modules/@thegetty/qu ...
+       ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (-S.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

### Expected behavior

`quire --version` should return "v1.0.0-rc.3`

### Version numbers

Windows 11
Quire CLI 1.0.0-rc.3
Node 18.12.1

**Comment by Erin-Cecele** [comment]
This is related to Jira ticket [DEV-14405](https://jira.getty.edu/browse/DEV-14405)

**Comment by Erin-Cecele** [comment]
Quire user @metro-glenn has gotten further along in the install process with Windows 11 than we have been able to reproduce. See Discussion #724 for more details.

**Comment by geealbers** [comment]
Happy to report that this issue is fixed by using node 18.16.0 (the current LTS version), and we've updated Quire to this version accordingly.
